test/gui/config.sample.ini

```diff
@@ -9,4 +9,5 @@ LOWEST_SYNC_TIMEOUT=
 CLIENT_LOG_FILE=
 TEMP_FOLDER_PATH=
 CLIENT_CONFIG_DIR=
-GUI_TEST_REPORT_DIR=
\ No newline at end of file
+GUI_TEST_REPORT_DIR=
+OCIS=false
```

Comment:
why we are assigning value for OCIS only?

Reply:
to set as default value and also it indicates that it will accept `true` or `false`.